Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -47,6 +47,8 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .productsOnboarding:
             return buildConfig == .localDeveloper || buildConfig == .alpha
+        case .productsPreview:
+            return buildConfig == .localDeveloper || buildConfig == .alpha
```

Comment:
I think we could use the same `.productsOnboarding` feature flag right? 
When the other deliverables are done, we can just take them out of the feature flag check.

Reply:
Right, updated in edf3702